lib/faraday/encoders/nested_params_encoder.rb

```diff
@@ -99,7 +99,8 @@ def self.decode(query)
           if !last_subkey || is_array
             value_type = is_array ? Array : Hash
             if context[subkey] && !context[subkey].is_a?(value_type)
-              raise TypeError, format("expected %s (got %s) for param `%s'", value_type.name, context[subkey].class.name, subkey)
+              raise TypeError, format("expected %<value_type_name>s (got %<class_name>s) for param `%<subkey>s'",
+                                      value_type_name: value_type.name, class_name: context[subkey].class.name, subkey: subkey)
```

Comment:
If this would go on more lines, each key-value pair in the hash on its own line, we could avoid adding new Metrics/LineLength issues.

Reply:
Thank you for your comments!
I try do it.